android/java/org/chromium/chrome/browser/crypto_wallet/util/Utils.java

```diff
@@ -1065,6 +1066,13 @@ public static AccountInfo findAccount(AccountInfo[] accounts, String address) {
         return null;
     }
 
+    public static List<AccountInfo> filterAccountsByCoin(
```

Comment:
nit: Can create an `AccountUtils` for this.

Reply:
Ok, in some next PR